examples/sink.exs

```diff
@@ -28,26 +29,25 @@ defmodule Example do
         hackney_opts: [follow_redirect: true]
       })
       |> child(:video_parser, %Membrane.H264.Parser{
-        generate_best_effort_timestamps: %{framerate: {25, 1}}
+        generate_best_effort_timestamps: %{framerate: {25, 1}},
+        output_stream_structure: :avc1
       })
       |> child(:video_realtimer, Membrane.Realtimer)
-      |> child(:video_payloader, Membrane.MP4.Payloader.H264)
       |> via_in(Pad.ref(:video, 0))
       |> get_child(:rtmp_sink),
       child(:audio_source, %Membrane.Hackney.Source{
         location: @audio_url,
         hackney_opts: [follow_redirect: true]
       })
       |> child(:audio_parser, %Membrane.AAC.Parser{
-        in_encapsulation: :ADTS,
         out_encapsulation: :none
       })
       |> child(:audio_realtimer, Membrane.Realtimer)
       |> via_in(Pad.ref(:audio, 0))
       |> get_child(:rtmp_sink)
     ]
 
-    {[spec: structure, playback: :playing], %{streams_to_end: 2}}
```

Comment:
Has something changed in the core that `playback: :playing` is no longer required?

Reply:
Indeed, it has been removed in https://github.com/membraneframework/membrane_core/releases/tag/v0.12.1

Reply:
Good to know :)